README.md

```diff
@@ -1,21 +1,25 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
+A jquery method that tells the app how to behave on a click.
 
-(b) Where in your JS file should you put it, and WHY? 
+(b) Where in your JS file should you put it, and WHY?
+  near thre bottom (but not necessarily).  Because you should be declaring variables near the top of the file.
```

Comment:
Your answer is incomplete - click handlers must go in the `$(document).ready()` assuming you want them to work as soon as the page loads.